src/TheThingsNetwork.cpp

```diff
@@ -44,11 +44,12 @@ bool TheThingsNetwork::sendCommand(String cmd) {
 }
 
 bool TheThingsNetwork::sendCommand(String cmd, String value) {
-  int l = value.length();
-  byte buf[l];
-  value.getBytes(buf, l);
+  String str = "";
```

Comment:
I figured that we are sending Strings now and we didn't need to convert them in bytes, I wasn't sure about this... Should I create another sendComand() ?